transport/session/src/manager.rs

```diff
@@ -696,6 +698,7 @@ impl<S: SendMsg + Clone + Send + Sync + 'static> SessionManager<S> {
         data: ApplicationData,
     ) -> crate::errors::Result<DispatchResult> {
         if (0..self.cfg.session_tag_range.start).contains(&data.application_tag) {
+            // TODO(20250520): modify range to have start from another distinct lower value
```

Comment:
Is this TODO resolved ?

Reply:
Yes, I modified the session management code to reflect the passthrough nature of new tagging mechanism.